docs: dynamic host volume command line (#24814)
hashicorp/nomad: website/content/docs/commands/volume/create.mdx

the [Controller][csi_plugins_internals] interface. The `volume create` command
automatically [registers][] the volume as well.

## Usage

```plaintext
nomad volume create [options] [file]
```

The `volume create` command requires a single argument, specifying the path to a
file containing a valid [volume specification][volume_specification]. Nomad
reads the file and submits the volume to the server for placement. If the
supplied path is "-", the volume file is read from STDIN. Otherwise the file is
read from the file at the supplied path.

When ACLs are enabled, this command requires a token with the appropriate
capability in the volume's namespace: the `csi-write-volume` capability for CSI
volumes or `host-volume-create` for dynamic host volumes.

## General Options

@include 'general_options.mdx'

## Volume Create Options

- `-detach`: Return immediately instead of entering monitor mode for dynamic
  host volumes. After creating a volume, Nomad prints the volume ID to the
  screen, which you can use to examine the volume using the `volume status`
  command. If `-detach` is omitted or false, the command monitors the state

>   of the volume until it is ready to be scheduled. Not valid for CSI volumes.

Author: Will add "has been fingerprinted by the client" to clarify this.

The [`volume create`][] command can create dynamic host volumes if host volume
plugins are installed on the node, or CSI volumes if the CSI plugins implement
the [Controller][csi_plugins_internals] interface. The `volume create` command
automatically [registers][] the volume as well.

## Usage

```plaintext
nomad volume create [options] [file]
```

The `volume create` command requires a single argument, specifying the path to a
file containing a valid [volume specification][volume_specification]. Nomad
reads the file and submits the volume to the server for placement. If the
supplied path is "-", the volume file is read from STDIN. Otherwise the file is
read from the file at the supplied path.

When ACLs are enabled, this command requires a token with the appropriate
capability in the volume's namespace: the `csi-write-volume` capability for CSI
volumes or `host-volume-create` for dynamic host volumes.

## General Options

@include 'general_options.mdx'

## Volume Create Options

- `-detach`: Return immediately instead of entering monitor mode for dynamic
  host volumes. After creating a volume, Nomad prints the volume ID to the

>   screen, which can be used to examine the volume using the `volume status`

Reviewer: ```suggestion
  screen, which you can use to examine the volume using the `volume status`
```

>   command. If `-detach` is omitted or false, the command will monitor the state

Reviewer: ```suggestion
  command. If `-detach` is omitted or false, the command monitors the state
```

  of the volume until it is ready to be scheduled. Not valid for CSI volumes.

- `-verbose`: Display full information when monitoring volume state. Used for
  dynamic host volumes only. Not valid for CSI volumes.

- `-policy-override`: Sets the flag to force override any soft mandatory
  Sentinel policies. Used for dynamic host volumes only. Not valid for CSI
  volumes.

## Volume Specification

> The volume specification for CSI is documented in the [CSI Volume
> Specification][csi_vol_spec] page. The volume specification for dynamic host
> volumes is documented in the [Dynamic Host Volume Specification](host_vol_spec)
> page.

Reviewer: ```suggestion
Refer to the [CSI Volume Specification][csi_vol_spec] and the [Dynamic Host Volume Specification](host_vol_spec) for further information.
```

active voice and brevity
